### PR TITLE
gradle minor improvements, support windows

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -21,13 +22,13 @@ def versionMinor = 8
 def versionPatch = 0
 def versionModifier = ""
 def jarVersion = "2.8.0"
-def secureEnv = System.getenv("TRAVIS_SECURE_ENV_VARS") != null ? System.getenv("TRAVIS_SECURE_ENV_VARS") : "false";
-def branch = System.getenv("TRAVIS_BRANCH") != null ? System.getenv("TRAVIS_BRANCH") : "";
-def tag = System.getenv("TRAVIS_TAG") != null ? System.getenv("TRAVIS_TAG") : "";
-def useSigning = "true".equals(secureEnv) && ("master".equals(branch) || (!"".equals(tag) && tag.equals(branch))) // tag will be the branch on a tag event for travis
+def secureEnv = System.getenv("TRAVIS_SECURE_ENV_VARS") != null ? System.getenv("TRAVIS_SECURE_ENV_VARS") : "false"
+def branch = System.getenv("TRAVIS_BRANCH") != null ? System.getenv("TRAVIS_BRANCH") : ""
```
Comment (scottf) [inline build.gradle:26]: groovy does not require semi-colons

File: build.gradle [hunk 2]
```diff
@@ -21,13 +22,13 @@ def versionMinor = 8
...
+def tag = System.getenv("TRAVIS_TAG") != null ? System.getenv("TRAVIS_TAG") : ""
+def useSigning = "true" == secureEnv && ("master" == branch || ("" != tag && tag == branch)) // tag will be the branch on a tag event for travis
```
Comment (scottf) [inline build.gradle:28]: groovy treats == like equals, not like in java as object equivalency

File: build.gradle [hunk 3]
```diff
@@ -134,6 +135,7 @@ javadoc {
     excludes = ['io/nats/client/impl', 'io/nats/examples']
     classpath = sourceSets.main.runtimeClasspath
     doLast {
+        if (!OperatingSystem.current().isWindows()) {
```
Comment (scottf) [inline build.gradle:138]: this allows the build to work on windows while not affecting any other operating system

**Review by ColinSullivan1 — APPROVED** [review]
LGTM - thanks!

**Pull request merged**